src/jit/lsraxarch.cpp

```diff
@@ -2372,6 +2372,44 @@ int LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
         // must be handled within the case.
         switch (intrinsicId)
         {
+            case NI_Base_Vector128_CreateScalarUnsafe:
+            case NI_Base_Vector128_ToScalar:
+            case NI_Base_Vector256_CreateScalarUnsafe:
+            case NI_Base_Vector256_ToScalar:
+            {
+                assert(numArgs == 1);
+
+                if ((baseType == TYP_FLOAT) || (baseType == TYP_DOUBLE))
+                {
+                    // We will either be in memory and need to be moved
```

Comment:
This still isn't perfect as the JIT often decides that `op1Reg` and `tgtReg` should not be the same.

This could probably be improved by allowing these to be contained and handling them in codegen; but I'm hoping there might be a better/more general-purpose solution...

Reply:
(Same goes for `ToVector256Unsafe` and `GetLower` below)